CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 Here we document changes that affect the public API or changes that needs to be communicated to other developers. 
 
+## 2024-12-09 string_view class identifiers
+All base classes that has a `virtual std::string getClassIdentifier() const` are refactored into 
+`virtual std::string_view getClassIdentifier() const`. This affects all classes derived from `Inport`, `Outport`, `Property`, `Metadata`, `Camera`,`animation::Track`, `animation::Interpolation`. This is done to avoid string allocations. 
+The trait classes `DataTraits` and `PortTraits` now uses `static constexpr std::string_view classIdentifier()` by default. 
+Look at `inviwo/core/datastructures/data/datatraits.h` for more information.
+
```

Comment:
```suggestion
## 2024-12-09 __Breaking__ change: string_view class identifiers
All base classes that have a `virtual std::string getClassIdentifier() const` were refactored into 
`virtual std::string_view getClassIdentifier() const`. This affects all classes derived from `Inport`, `Outport`, `Property`, `Metadata`, `Camera`, `animation::Track`, `animation::Interpolation`. This is done to avoid string allocations. 
The trait classes `DataTraits` and `PortTraits` now use `static constexpr std::string_view classIdentifier()` by default. 
Look at `inviwo/core/datastructures/data/datatraits.h` for more information.

```
```

Reply:
Are there changes necessary to existing code? If yes, that should be mentioned.